website/client/components/groups/tavern.vue

```diff
@@ -764,6 +782,22 @@ export default {
         this.$root.$emit('bv::show::modal', 'world-boss-rage');
       }
     },
+    async viewStaffProfile (staffId) {
+      const result = await this.$store.dispatch('members:fetchMember', { memberId: staffId });
```

Comment:
This looks good but I think the code an be simplified, using something like this

```
    async clickMember (hero) {
      let heroDetails = await this.$store.dispatch('members:fetchMember', { memberId: hero._id });
      this.$root.$emit('habitica:show-profile', {
        user: heroDetails.data.data,
        startingPage: 'profile',
      });
    },
```

handling errors is not necessary as we have interceptors that manage them in case something doesn't work